src/pages/Checkout.vue

```diff
@@ -231,7 +231,8 @@ export default {
           },
           shipping_method_code: this.shipping.shippingMethod,
           shipping_carrier_code: this.shipping.shippingMethod,
-          payment_method_code: this.payment.paymentMethod
+          payment_method_code: this.payment.paymentMethod,
+          shippingExtraFields: this.shipping.extraFields
```

Comment:
Adding the extra shipping fields to the data beeing sent to the api.

Unsure whether to use camel or snake case as its unclear from the extisting attributes